cluster-autoscaler/utils/drain/drain_test.go

```diff
@@ -489,6 +509,16 @@ func TestDrain(t *testing.T) {
 			expectBlockingPod:   &BlockingPod{Pod: emptydirPod, Reason: LocalStorageRequested},
 			expectDaemonSetPods: []*apiv1.Pod{},
 		},
+		{
```

Comment:
I'd consider also including test cases for pod with local volume(s) and:
* The annotation value being empty (=no local volumes match, pod is blocking)
* The annotation value not matching any local volumes on the pod (=pod is blocking)
* The annotation value only matching one local volume on the pod, but there is another one without the annotation (=pod is still blocking)
* [if we have multiple values available] Multiple annotation values matching all local volumes on the pod (=pod is not blocking)

Reply:
All of them make sense. I came up with a couple more after reading your comment.

Reply:
I have added the new test cases. Thank you.